plugins/MobileAppType/lang/en.json

```diff
@@ -0,0 +1,7 @@
+{
+  "MobileAppType": {
+    "MobileApp": "Mobile App",
+    "MobileApps": "Mobile Apps",
+    "MobileAppDescription": "A native mobile app for eg iOS, Android, .."
```

Comment:
I'd rather use something like `A native mobile app for iOS, Android or any other mobile operating system`. `Titanium Mobile` is a framework but not an operating system. There are many other mobile platforms out there so it's not only limited to iOS & Android. Just because we only have SDKs for them doesn't mean they are not tracking something else :) Also we wouldn't have to update the description when there are more SDKs